src/1_utils.js

```diff
@@ -54,6 +54,21 @@ utils.userPreferences = {
 		// Used by 3_api.js to determine whether a request should be blocked
 		var urlParser = document.createElement('a');
 		urlParser.href = url;
+
+		// INTENG-11512
+		// To allow SMS when tracking disabled, we must allow GET <actual link>.
+		// This precludes a filter on the path. Only apply the whitelist to
+		// service endpoints.
+		var whiteListDomains = [ config.api_endpoint, config.app_service_endpoint, config.link_service_endpoint ];
```

Comment:
there's already some of this logic for the api and app service here https://github.com/BranchMetrics/web-branch-deep-linking-attribution/blob/cdfcdc9a1c2fa743f257e432429a3b0b8d19b6db/src/1_utils.js#L46

and it's used on line 87 `var whiteListedEndpointWithData = utils.userPreferences.whiteListedEndpointsWithData[urlPath];`

can we use the same object and add the link service endpoint and check the path in that manner that way the `c` only gets hardcoded in one place?

Reply:
ahhh but the issue is we need a domain check... I see,  would simply adding `/c/` there not work?